src/templates/finops-hub/schema/schema_ea_normalized.json

```diff
@@ -0,0 +1,723 @@
+{
+  "additionalColumns": [
+    {
+      "name": "costInPricingCurrency",
+      "value": "$$COLUMN:CostInBillingCurrency"
+    },
+    {
+      "name": "costInUsd",
+      "value": 0.0
+    },
+    {
+      "name": "exchangeRateDate",
+      "value": "$$COLUMN:Date"
+    },
+    {
+      "name": "exchangeRatePricingToBilling",
+      "value": 1.0
+    },
+    {
+      "name": "invoiceId",
+      "value": ""
+    },
+    {
+      "name": "location",
+      "value": "$$COLUMN:ResourceLocation"
+    },
+    {
+      "name": "paygCostInBillingCurrency",
+      "value": "$$COLUMN:BillingCurrencyCode"
+    },
+    {
+      "name": "paygCostInUsd",
+      "value": 0.0
```

Comment:
Same as costInUsd. Null might be better.

Reply:
Done